website_product_availability/views/product_view.xml

```diff
@@ -22,5 +22,36 @@
                 </xpath>
             </field>
         </record>
+
```

Comment:
Dude, the two new trees you created are one in the upper side and one in the down side, can you put them at the same level, I mean like in the same line:
![image](https://www.evernote.com/l/AFxkEAlgkw5BXonJrWNvivIRnjrRL28VFeoB/image.png)